count/View/DeterminiteProgressView.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+struct DeterminateProgressView: View {
```

Comment:
I implemented this view similarly to the earlier `IndeterminateProgressView`. Both _could_ be implemented as `ProgressViewStyle` conforming types, but at this point I think they're more straightforward to read as simple `View`s.